.gitignore

```diff
@@ -15,3 +16,5 @@ decko*gem
 *.DS_Store
 *.swp
 test_work/*
+*.orig
+card/spec/config/bucket_credentials.yml
```

Comment:
safer to say something like **/bucket_credentials.yml?  or is there a template that this would cause problems for?